.github/workflows/on-pull-request.yml

```diff
@@ -4,6 +4,9 @@ on:
   pull_request:
     branches: [ master ]
 
+env:
+  GOLANGCI_LINT_VERSION: v1.54.2
```

Comment:
Latest is now v1.56.2.

Reply:
Please also update Makefile:  https://github.com/mailgun/gubernator/blob/master/Makefile#L5

Reply:
done 👍